files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html

```diff
@@ -229,13 +229,6 @@ <h2 id="Browser_compatibility" name="Browser_compatibility">ブラウザーの
 
 <p>{{Compat("javascript.builtins.Intl.DateTimeFormat.DateTimeFormat")}}</p>
 
-<h3 id="Implementation_Progress" name="Implementation_Progress">実装の進捗状況</h3>
-
-<p>以下の表は、クロスブラウザーの安定性にまだ達していない新機能の日々の実装状況を示しています。このデータは、 JavaScript の標準テストスイートである <a href="https://github.com/tc39/test262">Test262</a> で関連する機能テストをナイトリービルド、または各ブラウザの JavaScript エンジンの最新リリースで実行することで生成されます。</p>
-
-<div>{{EmbedTest262ReportResultsTable("Intl.DateTimeFormat-datetimestyle")}}</div>
-</div>
```

Comment:
Here you've deleted 2 `</div>` but only `<div>`.
Is this page working now? Was it always busted perhaps? Can you take a look?

Reply:
I tested http://localhost:5000/ja/docs/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat and it's looking fine.

Reply:
I'm going to check the file for the number of times the <div> are opened anyway to make sure it's not just the automatic closing of the browser that would give a false positive

Reply:
Founded a unused `<div>` on the top of the content, it's ok now ^^